jwst/ami/nrm_core.py

```diff
@@ -165,6 +153,10 @@ def fit_fringes_single_integration(self, slc):
         """
         Generate the best model to match a single slice.
 
+        TODO: Is it a bug that the LgModel is being initialized with the
+        zeroth wavelength every time?  When we iterate over this, it's an iteration over
+        wavelength.
```

Comment:
@rcooper295 Can you comment on whether it is a bug or not that when a single slice is fitted, it's always using `bandpass=self.instrument_data.wls[0]`, but [this line](https://github.com/spacetelescope/jwst/blob/7a3fc5f3667eec4dffe6a4ba7cd76fd9e1c28122/jwst/ami/nrm_core.py#L101) in `fit_fringes_all` appears to call this in a `for` loop iterating over wavelength?

Reply:
The above comment is not correct - no iteration over wavelength was being done.  The accessing of `instrument.data.nwav` in the linked line was simply being used to get the number of slices, since `nwav == nslices`.  This was changed to use `nslices` to clarify the intent of the loop, which is to iterate over slices.